src/common/modal/index.jsx

```diff
@@ -24,11 +29,17 @@ const Modal =({ title, show, onClose, onSubmit, children, cname })=> {
         <div className={`modal-${cname}-header`}>
           <h2 className="modal-title">{ title }</h2>
         </div>
+        {/* Clear All filters button */}
+        <div className={classes["modal-clear-filter"]}>
+          <button 
```

Comment:
Make it a link

Reply:
Okay! Will make it a link

Reply:
![image](https://user-images.githubusercontent.com/82811112/191246111-c5f2c468-8dc9-452a-a143-4006ff7110d0.png)

Have a look.

Reply:
my mouse pointer got hidden while taking the screenshot
Underline is shown when hovered on "Clear All"